client/components/ZoopDetails.tsx

```diff
@@ -43,13 +54,28 @@ const ZoopDetails = ({ zoop }: ZoopProps) => {
             </IconButton>
           </Stack>
         </Stack>
-        <Card>
-          <CardContent>
-            <Stack direction="row" justifyContent="center" alignItems="start">
-              <Typography variant="body1">{zoop.content}</Typography>
-            </Stack>
-          </CardContent>
-        </Card>
+
+          <Card>
+            <CardContent>
+              {currentUser && currentUser.id === zoop.authorId && (
+                <Stack direction="row" justifyContent="end">
+                  <ButtonGroup>
+                    <UpdateZoopButton 
+                      zoop={zoop}
+                    />
+                    </ ButtonGroup>
+                </Stack>
+              )}
+              <Link
```

Comment:
I don't know if link should be here? Then even on a zoop's own page it will be a link for itself

@jvaneyken to make sure you review this piece as well

Reply:
Yeah, we wouldn't want that.